content/docs/1.0.0/tutorial/create-your-first-microservice-javascript.md

```diff
@@ -20,29 +20,21 @@ To keep things fairly simple, we'll want to have our microservice do something i
 
 So let's build a microservice that computes a real-time [moving average](https://wikipedia.org/wiki/Moving_average) from the input, sending the result to any other microservices that might be listening. We'll also set it up to use dynamic configuration for how long the rolling window should be for, so that we can change it later without needing to restart anything.
```

Comment:
Let's build a microservice that computes a real-time [moving average](https://wikipedia.org/wiki/Moving_average) from the input, which will send the result to any other microservices that might be listening. We'll also set up a configuration to dynamically change the duration of the rolling window so that we can change it later without needing to restart anything.